docs/recipes/gulp/gulpfile.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const gulp = require('gulp');
+const connect = require('gulp-connect');
+const lighthouse = require('../../../lighthouse-core');
+const perfConfig = require('../../../lighthouse-core/config/perf.json');
+const port = 8080;
+
+const connectServer = function() {
+  return connect.server({
+    root: '../public',
+    livereload: true,
+    port: port
+  });
+};
+
+function handleOk() {
+  connect.serverClose();
+  process.exit(0);
```

Comment:
I thought that all LH log will be shown in a console.
E.g. in `pwmetrics` for `pass/fail` we use smth like `smokehouse`.
What would you suggest? 
Maybe reviewer (if we talking about project where LH will be injected) will look at CI and say it has bad trace, but 😒 , who does that..., so I dunno... 
But I think bringing smokehouse as user feature will be kinda overhead...